### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by talimarcus**
@davidrleonard do you mind taking a look at this?

### CODE REVIEW

**Comment by mdwragg** [comment]
I'll do it @talimarcus @davidrleonard

File: README.md [hunk 1]
```diff
@@ -1,5 +1,127 @@
...
+
+### Running a local build
+
+When you think you are ready to deploy your changes, you can build the site locally to test how the changes will behave in production. The localBuild task runs additional tasks that simulate how the site will be built for production. Many of these tasks take a significant amount of time to run, so you should only run this command once you want test that your changes are ready to push to production.
+
+To run a local build:
+  ```
+  $ gulp localBuild
+  $ cd build/default
+  $ npm serve
```
Comment (mdwragg) [inline README.md]: just `serve` (or any other static web server). You'll need to explain how to install it above, though `npm -g i serve` etc.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,127 @@
...
+  ```
+
+### Running a production build
+You should never run the `gulp prodBuild` task manually. This task is run on Travis as part of our CI/CD process when a change is pushed to production.
+
+## Build a new component
+
+### Developing the component
```
Comment (mdwragg) [inline README.md:53]: Not sure this section belongs in this repo?

File: README.md [hunk 3]
```diff
@@ -1,5 +1,127 @@
...
+All components should have an interactive documentation and demo page on the [Predix UI catalog](https://www.predix-ui.com). These documentation pages are built automatically by Travis CI for each component when a new release (i.e. a new tag) is pushed to Github.
+
+Before you begin, you'll need to install the following tools:
+
+1. [Ruby](https://www.ruby-lang.org/en/) version 1.9.3 or higher (this could be tricky, and the ruby build that shipped with your system might be too old)
+2. [Travis CLI ruby gem](https://github.com/travis-ci/travis.rb)
+
+You should also do the following before starting:
+
+1. Make sure you switch off of BLUESSO (or any proxied Wifi connection) onto Internet (or any open Wifi connection). You should also turn off your VPN, if you have one.
```
Comment (mdwragg) [inline README.md]: I do all this while on the corporate ethernet - not sure this advice is universal.

File: README.md [hunk 4]
```diff
@@ -1,5 +1,127 @@
...
+### Adding the component to site nav and component gallery
+
+1. Add the component to `elements/px-catalog/pages.json`.
+    ```
+    {
+      "path": "px-COMPONENT-NAME",
+      "label": "COMPONENT NAME",
+      "module": "px-COMPONENT-NAME-demo",
+      "entrypoint": "/bower_components/px-COMPONENT-NAME/demo/px-COMPONENT-NAME-demo.html",
+      "redirects": [
```
Comment (mdwragg) [inline README.md]: @davidrleonard - this is only for existing components, right? New stuff doesn't need this?

File: README.md [hunk 5]
```diff
@@ -1,5 +1,127 @@
...
+
+    These images will be compressed and put in the `img/component-gallery` directory.
+
+    **Note: If the component should not be shown in the component gallery, add it to the 'ExcludedComponentsArray' in `scripts/json-builder/create-components-info.js`**
+3. Follow the instructions in the [Running a local build](#running-a-local-build) section above to test your changes before pushing them to production.
+
+
+## Support and further information
+
+You can ask questions and get support on the [Predix Forum](https://www.predix.io/community). If you find any bugs, have issues with the code, or want to request enhancements, file a issue on the [Github repo](https://github.com/predix-ui/predix-ui.github.io).
```
Comment (mdwragg) [inline README.md]: Can probably drop the predix.io section...

Comment (mdwragg) [reply README.md]: But fine to leave the github bit.

**Review by mdwragg — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (davidrleonard) [reply README.md]: Yep redirects only for existing i think.

**Pull request merged**